.github/ISSUE_TEMPLATE/refactor_tickets.md

```diff
@@ -0,0 +1,20 @@
+---
+name: 🔧 Refactor ticket
+about: Help us improve our code by refactoring it.
+labels: refactor
+title: 🔧 [Refactor]
+---
+
+## :zap: Refactor ticket
+<!-- Describe your issue in detail. Include screenshots if needed. Give us as much information as possible. Use a clear and concise description of what the problem is.-->
+
+### Motive
+<!-- What is the purpose of this refactoring? If it's removing depcrecated code, please link to the deprecation notice. -->
+### Related issues
+<!-- Are there any related issues to link to? Please paste them below for reference. -->
+
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-02-04T14:25:18Z" title="Tuesday, February 4th 2020, 3:25:18 pm +01:00">Feb 4, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,18 @@
+---
+name: 🐛 Refactor ticket
+about: Help us improve our code by refactoring it.
+labels: refactor
+title: 🐛 [Refactor] 
+---
+
+## :zap: Refactor ticket
+<!-- Describe your issue in detail. Include screenshots if needed. Give us as much information as possible. Use a clear and concise description of what the bug is.-->
+
+### Motive
+<!-- What is the purpose of this refactoring? If it's removing depcrecated code, please link to the deprecation notice. -->
+
```


Would a section `Related issues` be helpful, to point out where this came up? Or is it too much? I usually add it under `Additional context` when its relevant but know that not everyone does. :smile:

I'm ok with it either way.